### ISSUE TITLE
:__init__() missing 1 required positional argument: 'enable_onnx_checker'跑官方自动压缩出错

### ISSUE THREAD

**Issue by arya-STARK-Z** [PaddlePaddle/X2Paddle#938]
-----------  Running Arguments -----------
 Distillation:
         alpha: 1.0
         loss: soft_label
 Global:
         arch: YOLOv7
         coco_dataset_dir: E://hj2//ex//shijuedatasetuse//COCO//
         coco_train_anno_path: annotations_trainval2017//annotations//instances_train2017.json
         coco_train_image_dir: train2017//train2017
         coco_val_anno_path: annotations_trainval2017//annotations////instances_val2017.json
         coco_val_image_dir: val2017//val2017
         image_path: None
         model_dir: E://hj2//ex//shijue3//yolov7men//yolov7ori//yolov7//yolov7-tiny.onnx
 QuantAware:
         activation_quantize_type: moving_average_abs_max
         onnx_format: True
         quantize_op_types: ['conv2d', 'depthwise_conv2d']
 TrainConfig:
         eval_iter: 1000
         learning_rate:
                 T_max: 8000
                 learning_rate: 3e-05
                 type: CosineAnnealingDecay
         optimizer_builder:
                 optimizer:
                         type: SGD
                 weight_decay: 4e-05
         train_iter: 5000
------------------------------------------
loading annotations into memory...
Done (t=16.47s)
creating index...
index created!
loading annotations into memory...
Done (t=0.52s)
creating index...
index created!
2023-02-10 01:48:50,901-INFO: Now translating model from onnx to paddle.
2023-02-10 01:48:50,902-WARNING: __init__() missing 1 required positional argument: 'enable_onnx_checker'
2023-02-10 01:48:50,902-ERROR: x2paddle threw an exception, you can ask for help at: https://github.com/PaddlePaddle/X2Paddle/issues
![image](https://user-images.githubusercontent.com/61528997/217897434-42c74ac0-7765-43c5-83a5-4905876d2825.png)
跑的官方yolov7的示例
本示例启动自动压缩以YOLOv7-Tiny为例，如果想要更换模型，可修改--config_path路径即可，具体运行命令为：

单卡训练：
export CUDA_VISIBLE_DEVICES=0
python run.py --config_path=./configs/yolov7_tiny_qat_dis.yaml --save_dir='./output/'

**Comment by wjj19950828** [comment]
@arya-STARK-Z hi，目前问题已经修复，输入下列命令，安装最新版本X2Paddle

```shell
## 多pip uninstall几次，确保老版本x2paddle卸载干净
pip uninstall x2paddle
## 安装develop版本
git clone https://github.com/PaddlePaddle/X2Paddle.git
cd X2Paddle
python setup.py install
```
